src/test/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerMaintainLinkCaseTest.java

```diff
@@ -59,26 +69,31 @@ void shouldNotConfigureMaintainLinkCaseIfFeatureFlagFalse() {
     }
 
     @Test
-    void shouldAdd2LinksToCase() {
+    void shouldSuccessfullyStayTheCase() {
```

Comment:
are we using appropriate method name here?

Reply:
Updated the name.